doc/source/admin/nginx.md

```diff
@@ -130,10 +130,6 @@ http {
         }
 
 		# serve framework static content
-		location /static/style {
-			alias $galaxy_root/static/style/blue;
-			expires 24h;
-		}
 		location /static {
```

Comment:
Not connected to this PR, but I noticed that the code here is mixing spaces and tabs: convert all tabs to spaces?

Reply:
@nsoranzo Sure, I'll take a look.

Reply:
(done)